### PR TITLE
Confirmation modal when removing an item

### PR DESCRIPTION
**Pull request by auterium**
This is the same as the PR I did for the master branch over the CRUD module

### CODE REVIEW

**Comment by lirantal** [comment]
needs someone to review + test

File: vertical-module/templates/client/controllers/_.client.controller.js [hunk 1]
```diff
@@ -25,18 +25,33 @@ angular.module('<%= slugifiedPluralName %>').controller('<%= classifiedPluralNam
...
+						$modalInstance.dismiss('cancel');
+					};
+				}
+			})
+			.result.then(function() {
+				if ( <%= camelizedSingularName %> ) {
+					<%= camelizedSingularName %>.$remove();
+
+					for (var i in $scope.<%= camelizedPluralName %> ) {
+						if ($scope.<%= camelizedPluralName %> [i] === <%= camelizedSingularName %> ) {
```
Comment (ilanbiala) [inline vertical-module/templates/client/controllers/_.client.controller.js]: Too many spaces, no?

Comment (auterium) [inline vertical-module/templates/client/controllers/_.client.controller.js]: Where? I just wrapped the already existing code within the .result.then(function(){ }); block, and all the spaces are actually tabs. If this is not the standard way of doing it, I'm ok following any other guidelines if there are. I'm new in the nodejs world (less than 2 months, though over 7 years in other technologies), so I'm not ocmpletely familiar with the standards yet.

Comment (ilanbiala) [inline vertical-module/templates/client/controllers/_.client.controller.js]: before `[i]` and before the `) {` at the end of the line.

**Comment by auterium** [comment]
@lirantal I've removed such spaces

**Comment by auterium** [comment]
Why was it closed without merge?

**Comment by ilanbiala** [comment]
@auterium too much different code now that the generator works with Mean 0.4.x and we need to add new subgenerators based on a better system. Feel free to help with subgenerators.

**Comment by codydaig** [comment]
@auterium We just upgraded the project to rely heavily on the main meanjs/mean repository. It actually performs a git clone from that repo during the main app generator now. Sub generators are starting to get built out. Like @ilanbiala said, if you want to help build sub generators, feel free! We would greatly appreciate the help.

**Pull request closed without merging**